api/bin/chainflip-ingress-egress-tracker/src/witnessing/state_chain.rs

```diff
@@ -556,7 +556,7 @@ mod tests {
 						deposit_address: polkadot_account_id,
 						amount: 100u128,
 						asset: cf_chains::assets::dot::Asset::Dot,
-						deposit_details: (),
+						deposit_details: 1,
```

Comment:
i think there is no (external chain) block number on the `DepositFinalised`/`DepositBoosted`, so it would be nice to have that in the deposit detail (or on the event)